plynx/service/users.py

```diff
@@ -19,15 +19,17 @@ def run_list_users():
         print(','.join(map(str, [user._id, user.username])))
 
 
-def run_create_user(username, password):
+def run_create_user(email, username, password):
```

Comment:
this function is used in the same file, you will need to modify it as well. Also another function is used in in cli.py file